resources/definitions/Think3Dim_T22_Max_Pro.def.json

```diff
@@ -0,0 +1,64 @@
+{

```

Comment:
Could you change the file name to be lowercase with underscores, like the rest?

(And be sure to also update the reference to it in the extruder definition then.)